.eslintrc.json

```diff
@@ -1,15 +1,22 @@
 {
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
```

Comment:
nitpick (you'll notice this is a pattern with my reviews...)

can we alphabetize these keys 🙏